webpack.config.js

```diff
@@ -42,7 +42,7 @@ function buildConfig({
         type,
       },
       assetModuleFilename: "[name][ext]",
-      chunkFormat: "module",
+      chunkFormat: false,
```

Comment:
This is why the final bundle includes `new URL("./", import.meta.url)`.

Setting it to false to disable the chunks feature of webpack (which is not used anyway)

Reply:
Good find! Thanks!